infrastructure/Scripts/keyvaultsecrets-encrypted.ps1

```diff
@@ -0,0 +1,85 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]$Subscription,
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]$VaultName,
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]$encryptionkey,
```

Comment:
How are we passing the key?

Reply:
Yes it will be a secret variable in ADO which we can pass as parameter to the script